Project2015To2017.Migrate2017.Tool/CommandLogic.cs

```diff
@@ -52,9 +52,11 @@ public CommandLogic()
 		public void ExecuteMigrate(
 			IReadOnlyCollection<string> items,
 			bool noBackup,
-			ConversionOptions conversionOptions)
+			ConversionOptions conversionOptions
+			)
```

Comment:
Code format.

Reply:
thanks, sorted now

Reply:
reverted now, I'm used to the symmetry of the opening and closing in ones like this, but didn't mean to leave in the change